pkgs/development/python-modules/yq/default.nix

```
@@ -14,27 +15,31 @@
 
 buildPythonApplication rec {
   pname = "yq";
-  version = "2.10.1";
+  version = "2.11.0";
 
-  propagatedBuildInputs = [ pyyaml xmltodict jq argcomplete ];
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1gp9q5w1bjbw7wmba5hm8ippwvkind0p02n07fqa9jlqglhxhm46";
+  };
+
+  propagatedBuildInputs = [
+    pyyaml
+    xmltodict
+    argcomplete
+  ];
 
   doCheck = true;
 
   checkInputs = [
    pytest
    coverage
    flake8
-   jq
+   pkgs.jq
    toml
   ];
 
   checkPhase = "pytest ./test/test.py";
```

```suggestion
  checkPhase = "pytest ./test/test.py";
  
  pythonImportsCheck =  [ "yq" ];
```

this will probably work with either Package or Application, but can't hurt